modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -236,65 +206,55 @@ INSTANTIATE_TEST_CASE_P(ConvertToFluid, ConvertToTest,
 INSTANTIATE_TEST_CASE_P(DISABLED_ConcatHorTestFluid, ConcatHorTest,
                         Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
                                 Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(-1),
                                 Values(CORE_FLUID)));
 
 INSTANTIATE_TEST_CASE_P(DISABLED_ConcatVertTestFluid, ConcatVertTest,
                         Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
                                 Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(-1),
                                 Values(CORE_FLUID)));
 
 INSTANTIATE_TEST_CASE_P(Split3TestFluid, Split3Test,
                         Combine(Values(CV_8UC3),
                                 Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(CV_8UC1),
                                 Values(CORE_FLUID)));
 
 INSTANTIATE_TEST_CASE_P(Split4TestFluid, Split4Test,
                         Combine(Values(CV_8UC4),
                                 Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(CV_8UC1),
                                 Values(CORE_FLUID)));
 
 INSTANTIATE_TEST_CASE_P(Merge3TestFluid, Merge3Test,
                         Combine(Values(CV_8UC1),
                                 Values(cv::Size(1920, 1080),
-                                       cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(CV_8UC3),
                                 Values(CORE_FLUID)));
 
 INSTANTIATE_TEST_CASE_P(Merge4TestFluid, Merge4Test,
                         Combine(Values(CV_8UC1),
                                 Values(cv::Size(1920, 1080),
-                                       cv::Size(1280, 720),
-                                       cv::Size(640, 480),
                                        cv::Size(128, 128)),
```

Comment:
Maybe we should declare the list of resolutions we test one and for all the tests? Then `ValuesIn(...)` could be used.

Reply:
Done